cli/cloe_launch/commands/deploy.py

```diff
@@ -0,0 +1,91 @@
+# Copyright 2023 Robert Bosch GmbH
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# SPDX-License-Identifier: Apache-2.0
+
+# pylint: disable=too-many-arguments
+
+"""
+Implementation for cloe-launch command deploy.
+
+Usage: cloe-launch [-v] deploy [-c] [-f] [-r] [-D PATH] CONANFILE [CONAN_INSTALL_ARGS]
```

Comment:
is this comment correct? in particular, i am wondering if "-c" and "-r" really exist, and whether --rpath/--no-rpath misses.

Reply:
Right you are.